docs/pages/versions/unversioned/sdk/app-integrity.mdx

```diff
@@ -0,0 +1,186 @@
+
+In simple terms, `@expo/app-integrity` helps your server tell the difference between:
+
+- Your **real app** running on a **real device**, and
+
+- Anything else (modified apps, scripts, emulators).
+
+It does this by using the platform-recommended app attestation services.
+
+## Installation
+
+<APIInstallSection />
+
+## Usage on iOS
+
+### Configuration
+
+In Xcode, go to **Signing & Capabilities**, click **+ Capability**, add **App Attest**. Xcode will automatically add the required entitlement to your app.
+
+> **Note** To use the App Attest service, your app must have an App ID that you register on the Apple Developer website.
+
+For verification logic on your server, see [Validating apps that connect to your server](https://developer.apple.com/documentation/devicecheck/validating-apps-that-connect-to-your-server).
+
+### Check if the device supports App Attestation
+
+Not all devices can use the App Attest service, so it’s important to have your app run a compatibility check before accessing the service. If the user’s app doesn’t pass the compatibility check, gracefully bypass the service. You check for availability by reading the `isSupported` property.
+
+```js
+import * as AppIntegrity from '@expo/app-integrity';
+
+if (AppIntegrity.isSupported) {
+  // Perform key generation and attestation.
+}
+// Continue with your server API access.
+```
+
+> **Note** App Attest is not supported on simulator.
+
+> **important**  Most app extensions don’t support App Attest. Generally, when executing code in these extensions, bypass key generation and attestation, even if the `isSupported` method property is `true`. The only app extensions that support App Attest are watchOS extensions in watchOS 9 or later. For these extensions, you can use the results from `isSupported` to indicate whether your WatchKit extension bypasses attestation.
+
+
+### Create a key pair
+
+For each user account on each device running your app, generate a unique, hardware-based, cryptographic key pair by calling the `generateKey` method.
+
+```js
+const keyId = await AppIntegrity.generateKey()
+```
+
+On success, the method returns a key identifier (`keyId`) that you use later to access the key. Record the identifier in persistent storage because there’s no way to use the key without the identifier, and no way to get the identifier later. The device automatically stores the associated private key in the Secure Enclave, from where the App Attest service can use it to create signatures, but from where no process can ever directly read or modify it, ensuring its security.
```

Comment:
```suggestion
On success, the method returns a key identifier (`keyId`) that you use later to access the key. Record the identifier in persistent storage because there's no way to use the key without the identifier and no way to get the identifier later. The device automatically stores the associated private key in the Secure Enclave, from where the App Attest service can use it to create signatures, but from where no process can ever directly read or modify it, ensuring its security.
```